tools/network-scene-vis.md

```diff
@@ -0,0 +1,50 @@
+---
+id: netscenevis
+title: Network Scene Visualization
+---
+
+The Network Scene Visualization (NetSceneVis) is a powerful, user-friendly tool included in the Multiplayer Tools package to help you visualize and debug network communication on a per-object basis in the scene view of your project with visualizations such as mesh shading and text overlay.
+
+The NetSceneVis tool listens to network messages about the game state to visually display bandwidth and ownership on a per-object basis in the Unity Editor. The visualizations are updated in real-time as the network messages are sent and received, showing the game state synchronization across the network.
+
+This tool can help optimize your network code to ensure that your game runs smoothly for all players.
```

Comment:
I think there's a missing "you" in this sentence after "help", and it should maybe read:
> This tool can help you optimize your network code to ensure that your game runs smoothly for all players.

The reason I say this is that the tool won't be optimizing the user's network code for them automatically 😅

Reply:
We could also mention that it could help them to debug too, as the tool is likely to be useful for that as well.